### PR TITLE
Edition of work creation hypotheses on an unbuilt node

### PR DESCRIPTION
**Pull request by anas-drissi**

### CODE REVIEW

**Review by AbdelHedhili — COMMENTED** [review]
i'm lost. why does the title of this PR is about hypotheses and then the code is all about security analysis and loadflow. What is the goal of this PR ?

**Review by bendaamerahm — COMMENTED** [review]
i think you need to change the title of this PR

**Comment by anas-drissi** [comment]
The title of the PR has been changed.

File: src/components/dialogs/input-hooks.js [hunk 1]
```diff
@@ -295,12 +295,11 @@ export const useConnectivityValue = ({
...
             voltageLevel: voltageLevelIdDefaultValue
-                ? voltageLevelOptions.find(
-                      (value) => value.id === voltageLevelIdDefaultValue
-                  )
+                ? {
+                      id: voltageLevelIdDefaultValue,
+                  }
                 : null,
             busOrBusbarSection: busOrBusbarSectionIdDefaultValue
                 ? {
```
Comment (AbdelHedhili) [inline src/components/dialogs/input-hooks.js:305]: what does this have to do with the badges ?

File: src/components/app-top-bar.js [hunk 2]
```diff
@@ -161,6 +179,24 @@ const AppTopBar = ({ user, tabIndex, onChangeTab, userManager }) => {
...
+        fetchLoadFlowInfos,
+        loadFlowStatusInvalidations
+    );
+
+    const [securityAnalysisStatusNode] = useNodeData(
+        studyUuid,
+        currentNode?.id,
+        fetchSecurityAnalysisStatus,
+        securityAnalysisStatusInvalidations
+    );
```
Comment (AbdelHedhili) [inline src/components/app-top-bar.js:199]: is there a reason why all this code is in the top-bar ?

**Review by AbdelHedhili — CHANGES_REQUESTED** [review]
